serverside/makoto/build.gradle.kts

```diff
@@ -51,6 +52,17 @@ val sourcesJar by tasks.registering(Jar::class) {
 }
 
 
+
+File("${project.layout.projectDirectory.dir("src")}/main/kotlin/wardenVersion.kt").writer().apply {
```

Comment:
This does work, but wouldn't be a separate directory for generated code better suited? Also the package name doesn't match the directory of the file.

Or use something like this https://github.com/gmazzo/gradle-buildconfig-plugin